nevergrad/functions/pyomo/core.py

```diff
@@ -175,3 +201,71 @@ def _pyomo_constraint_wrapper(self, i: int, instru: tp.ArgsKwargs) -> bool:
             return ret
         else:
             raise NotImplementedError(f"Constraint type {self.all_constraints[i].ctype} is not supported yet.")
+
+
+# Simple Pyomo models, based on https://www.ima.umn.edu/materials/2017-2018.2/W8.21-25.17/26326/3_PyomoFundamentals.pdf.
+def get_pyomo_list():
+    model = pyomo.ConcreteModel()
+    model.x = pyomo.Var([1, 2], domain=pyomo.NonNegativeReals)
+    model.obj = pyomo.Objective(expr=(model.x[1] - 0.5)**2 + (model.x[2] - 0.5)**2)
+    yield Pyomo(model)
+
+    # Rosenbrock --- pb with continuous variables! NotImplementedError: Cannot handle domain type <class 'pyomo.core.kernel.set_types.RealSet'>
+    # I don't get it, because it looks like the code above should accept bounded ranges.
+    rosenbrock = pyomo.ConcreteModel() 
+    #rosenbrock.x = pyomo.Var([1, 2], domain=pyomo.NonNegativeReals, bounds=(-2, 2))
+    rosenbrock.x = pyomo.Var(initialize=-1.2, bounds=(-2, 2)) 
+    rosenbrock.y = pyomo.Var(initialize=1.0, bounds=(-2, 2)) 
+    rosenbrock.obj = pyomo.Objective(expr=(1-rosenbrock.x)**2 + 100*(rosenbrock.y-rosenbrock.x**2)**2, sense=pyomo.minimize)
+    yield Pyomo(rosenbrock)
+
+    # Knapsack
+    for num_items in [4, 14, 44, 134]:
+        print(f"Creating Knapsack{num_items}")
+        if num_items == 4:
+            items = ['hammer', 'wrench', 'screwdriver', 'towel'] 
+            values = {'hammer':8, 'wrench':3, 'screwdriver':6, 'towel':11} 
+            weights = {'hammer':5, 'wrench':7, 'screwdriver':4, 'towel':3} 
```

Comment:
can you use a dict instead: {item: (value, weight)} should be simpler to read (way shorter)